validation/output.ts

```diff
@@ -0,0 +1,135 @@
+import {ComparisonResult, PageSummary} from './compare';
+import fs from 'fs';
+
+/**
+ * A preview for some test-case
+ */
+export interface CasePreview {
+  caseId: number;
+  // Less agreement groups --> more agreement amongst implementations.
+  // This property can be used to colour-code cases.
+  numAgreementGroups: number;
+  role?: string;
+}
+
+/**
+ * Creates a new test-case from a ComparisonResult and returns
+ * a CasePreview for that test-case.
+ * @param comparisonResult - The ComparisonResult that will become a test-case
+ */
+export function createTestcase(
```

Comment:
I think (throughout this file) `write` is more clear than `create` or `add` in function names